Tasks/Common/nuget-task-common/NuGetToolRunner2.ts

```diff
@@ -181,6 +209,27 @@ export function isCredentialProviderEnabled(quirks: NuGetQuirks): boolean {
     return true;
```

Comment:
Sorry if I wasn't clear before, but I thought we were going to disable the V1 provider for nuget >= 4.8.0. So if the nuget version matches 4.8.0, let's also log something like: "Credential provider V1 is disabled in favor of V2 plugin" and return false